src/components/NewApiDelegationPage/NewApiDelegationPage.module.css

```diff
@@ -0,0 +1,59 @@
+@media only screen and (min-width: 0px) {
+  .pageContainer{
+      --pageContainer-width: 100%;
+  }
+}
+
+@media only screen and (min-width: 700px) {
+  .pageContainer{
+    --pageContainer-width: 90%;
+  }
+}
+
+@media only screen and (min-width: 900px) {
+  .pageContainer{
+    --pageContainer-width: 60%;
+  }
+}
+
+.pageContent{
+  --component-page-margin-horizontal: 3rem;
+  margin-right: var(--component-page-margin-horizontal);
+  margin-left: var(--component-page-margin-horizontal);
+}
+
+.pageContainer{
+  width: var(--pageContainer-width);
+  margin: auto;
+  margin-top: 2rem;
+}
+
+.pageContentAccordionsContainer{
+    grid-template-columns: 1fr 1fr;
+    display:grid;
+}
+
+.apiAccordions{
+    margin-right: 3rem;
+}
+
+
+.searchField{
```

Comment:
Yeah I know. I'm not sure if it's smart to have it span half the page because it adds complexity because we have to handle it spanning to 100% on mobile screens and it's not really a problem for user to understand which list they are searching in

Reply:
Maybe ask UX? I would think there's a reason they've designed it that way.
Personally I think it looks neater with just half width. (I mean they have another design for small screens anyway)

Reply:
Either way, this Page is to be implemented in more than one iteration. We still have to wait for Multiselect to be implemented in DS and how Hanne decides it will look with Multiselect. So I think we can finish if it's going to span for 100% or half the page after Hanne has decided, and just follow her sketches in the next iteration when Multiselect is exported from DS.